src/mapkit.cs

```diff
@@ -1521,18 +1544,17 @@ partial interface MKMapSnapshot {
 		[Export ("pointForCoordinate:")]
 		CGPoint PointForCoordinate (CLLocationCoordinate2D coordinate);
 
-#if MONOMAC
 		[NoWatch][NoTV][NoiOS]
```

Comment:
Maybe a [NoMacCatalyst] should go here

Reply:
Nope, and xtro locally agrees. I had to remove some as they were used as a base class of something that is on catalyst.